### ISSUE TITLE
High reward then converged to low reward

### ISSUE THREAD

**Issue by r-lipton** [Unity-Technologies/ml-agents#348]
Hello,

I am trying to train an agent learning only from pixels (using only observation and no state, the actions space is discrete). 
However, I have a little problem, is it normal that the agent achieves high reward at the beginning, and then if I let it train a long time, it converged to a lower reward?
A simple solution would be to stop the training earlier, but is there a solution that does not require a manual check? Maybe learning rate decay? 
![tensorboard](https://user-images.githubusercontent.com/36620906/36369239-cf53897c-155a-11e8-8bb7-0330263c0063.PNG)

Thanks!

**Comment by r-lipton** [comment]
Maybe I have an explanation about that. 
It seems like the observation becomes a black image after some time.
For info, I train the agent on a remote server using these [instructions ](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Training-on-Amazon-Web-Service.md)

![capture](https://user-images.githubusercontent.com/36620906/36415760-52c4414e-1627-11e8-82ad-c534b4d78885.PNG)

When I restart the Xorg process, the observation returns the correct image.

**Comment by vincentpierre** [comment]
Can you try on your local machine to see if the agent learns differently ?

**Comment by r-lipton** [comment]
I did not try on my local computer, but I did save every frame for a while, and there was no problem. But the convergence was not good using the Unity PPO code (maybe wrong hyperparameters).
I tried an implementation of Atari DQN using default parameters and it worked :smiley:
